sklearn/ensemble/bagging.py

```diff
@@ -69,18 +95,15 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
         seed = random_state.randint(MAX_INT)
```

Comment:
Possibly they could, but I decided to leave it like it was so that I was changing as few lines as possible. Also I'd then need to pass the `starts` variable to `_parallel_build_estimators()` when its called within `BaseBagging._fit()`.

The result should be the same either way.